Calling/ClientApp/src/components/ChatHeader.tsx

```diff
@@ -0,0 +1,158 @@
+import {
```

Comment:
On screen, under header we have 2 buttons now for
- Leave
- Participant icon
- Settings
It will be a good idea to combine them as Video screen also had a header with these same buttons

Reply:
Created a separate task to track this suggestion.